content-test/selectors/selectors.test.js

```diff
@@ -117,5 +111,24 @@ describe("selectors", () => {
       const groups = [state.TopSites.rows, state.Spotlight.rows, state.TopActivity.rows];
       assert.deepEqual(groups, dedupe.group(groups));
     });
+    it("should sort TopActivity by dateLastVisited", () => {
+      state = selectNewTabSites({
+        TopSites: {rows: []},
+        Spotlight: {rows: []},
+        FrecentHistory: {rows: [
+          {url: "foo.com", lastVisitDate: 1},
+          {url: "bar.com", lastVisitDate: 4},
+          {url: "baz.com", lastVisitDate: 3}
+        ]}
+      });
+      console.log(state.TopActivity.rows);
```

Comment:
FWIW, Travis-CI caught it, but it's currently configured as an ESLint warning (not an error), so you'll only catch it if you are looking through Travis logs... 

https://travis-ci.org/mozilla/activity-streams/builds/114623632#L278

Let me know if you want to convert any ESLint rules from warnings=>errors or warnings=>ignore (or any other combination).